development/kops/eks-d.tpl

```diff
@@ -74,30 +73,14 @@ spec:
   kubeDNS:
     provider: CoreDNS
     coreDNSImage: {{ .coredns.repository }}:{{ .coredns.tag }}
-    externalCoreFile: |
```

Comment:
this was needed when we were using kops 1.18 and coredns 1.7. This is now the default in kops 1.19+

Reply:
I'm not sure where that came from